src/common/src/metrics/guarded_metrics.rs

```diff
@@ -208,6 +208,14 @@ impl<T: MetricVecBuilder, const N: usize> LabelGuardedMetricVec<T, N> {
         }
     }
 
+    pub fn reset(&self) {
+        self.inner.reset()
+    }
+
+    pub fn remove_label_values(&self, labels: &[&str; N]) -> prometheus::Result<()> {
```

Comment:
Why we still need `remove_label_values`? It is automatically handled when the labeled is not used any more.

Reply:
https://github.com/risingwavelabs/risingwave/blob/main/src/meta/src/hummock/metrics_utils.rs#L249
Since Meta handles the removal of Hummock metrics, maybe we should leave it untouched and change only the `source_is_up` that directly related to #12343?

Reply:
Previously the `reset` and `remove_label_values` are used because we were releasing the unused label manually. With the label guarded metrics, the labels will be automatically removed when the label is not used anymore. So we can either change only the metrics related to the issue, or we change some other current logic to remove the original calls on `reset` and `remove_label_values`.